misc/discoverynode/bin/run

```diff
@@ -11,8 +11,8 @@ ROOT_DIR=$(realpath $(dirname $0)/..)
 BASE_CONFIG=$(realpath $ROOT_DIR/etc/base_config.json)
 TMP_CONFIG="/tmp/discoverynode_config.json"
 
-if [[ $# -ne 3 ]]; then
-  error Usage: $0 SITE_MODEL TARGET DEVICE_ID
+if (( $# -le 4 )); then
```

Comment:
shouldn't this be  -le 2 since the [OPTIONS] bit is optional?  Or -lt 3?

Reply:
There's three required options: 
- site model
-  target 
- device ID